src/main/resources/application.yaml

```diff
@@ -44,6 +44,8 @@ storage:
   account_name: ${STORAGE_ACCOUNT_NAME:bulkscanning}
   key: ${STORAGE_KEY:dGVzdA==} #Base 64 encoded
   blob_lease_timeout: ${STORAGE_BLOB_LEASE_TIMEOUT:15}
+  signature_algorithm: ${STORAGE_BLOB_SIGNATURE_ALGORITHM:none}  # none or sha256withrsa
+  public_key_base64: ${STORAGE_BLOB_PUBLIC_KEY:dGVzdA==} #Base 64 encoded
```

Comment:
that public key is really short, how was it generated?

Reply:
It's not a public key. At this stage verification is disabled. For a real public key see the tests

Reply:
aah I see its `test` any reason its not empty?

Reply:
It's just to remind people that it needs to be base64 encoded. If you use that key you'll have failures anyway so it is as good as an empty one